c/ReachSafety-Combinations.cfg

```diff
@@ -0,0 +1,3 @@
+Description: Contains programs that are combinations of other programs. These programs contain program parts independent of each other and may require different techniques to be successfully verified.
```

Comment:
`.cfg` files do not exist anymore in the repository. A short description is part of the `.set` file, the rest in the readme. Architecture is replaced by the data model in task-definition files.

Reply:
Of course, thanks. I was only concerned with the license-stuff and didn't consider the tasks again, yet